src/Shared/XMakeAttributes.cs

```diff
@@ -94,26 +91,29 @@ internal struct MSBuildArchitectureValues
         /////////////////////////////////////////////////////////////////////////////////////////////
         internal const string defaultXmlNamespace = "http://schemas.microsoft.com/developer/msbuild/2003";
 
+        private static readonly HashSet<string> KnownSpecialTaskAttributes = new HashSet<string> { condition, continueOnError, msbuildRuntime, msbuildArchitecture, xmlns };
+
+        private static readonly HashSet<string> KnownSpecialTaskAttributesIgnoreCase = new HashSet<string>(StringComparer.OrdinalIgnoreCase) { condition, continueOnError, msbuildRuntime, msbuildArchitecture, xmlns };
```

Comment:
Could you prepend the fields with "_" to make them consistent with most of msbuild?

Reply:
`static readonly` members should look like fields and properties with no leading underscore and proper case.